### DISCUSSION TITLE
Masa Docker Image

### DISCUSSION THREAD

**Discussion by JasPanesar** [MasaCMS/MasaCMS#38] [Ideas]
Hi folks,

I quickly searched for a discussion around a Masa Docker Image.

Is this something that might be of interest to anyone? I used the Mura one quite regularly and there shouldn't be a change that big.  Happy to chat and collaborate.

Jas

**Comment by JasPanesar** [comment]
I have answered my own question - For anyone else who may be looking - docker details are in the new documentation that's under development at https://docs.masacms.com/getting-started/installation/local-docker/

**Reply by guustnieuwenhuis** [reply]
Hey Jas,
This Docker setup is for local dev environments. We advise against running that in production. I just want to make sure you understand that.
